hittingEndPoint.vb

```diff
@@ -173,4 +161,28 @@ Module hittingEndPoint
         Return lines
     End Function
 
+    Private Function readFileData() As String
+        Dim FileData As String = String.Empty
+        Dim SReader As IO.StreamReader = Nothing
+        Try
+            'Reading the file which was download after hitting the Endpoint.
+            SReader = New IO.StreamReader("D:\Json.txt")

```

Comment:
Create a global constant for "D:\Json.txt" since this is used at multiple places.

Reply:
Done.